### PR TITLE
DDS-917 Add toggle component

### PR DESCRIPTION
**Pull request by rinjInTokyo**
This PR adds Toggle (`daikin-toggle`) component.

## Related Issue(s) and Links

- JIRA Ticket (parent): [DDS-917](https://daikinsvoil.atlassian.net/browse/DDS-917)
- Figma design: [Figma](https://www.figma.com/design/VyaaU8Ta9yzyf0PsURWSSf/branch/wHAhw4yC54Hjq6Whb8FL5T/DDS%3A-Design-Kit?node-id=23902-559&m=dev)

## API Design
[JIRA Wiki](https://daikinsvoil.atlassian.net/wiki/spaces/GDS/pages/3097624601/Toggle+Component+API)

## How Has This Been Tested?

- VRT (Visual Regression Test) for [default, small] size and ["enabled", "disabled",] variant and ["checked", "unchecked",] state.
- Toggle will be checked if click on it and will be unchecked click again
- Toggle's check state will not be changed when disabled and click change event will not be fired

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have added appropriate VRTs.
- [x] I have added appropriate unit tests.
- [x] I have confirmed that my changes do not generate new warnings and errors.

[DDS-917]: https://daikinsvoil.atlassian.net/browse/DDS-917?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/toggle/daikin-toggle.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+  /**
+   * Specify whether the Toggle is in a error state
+   */
+  @property({ type: Boolean, reflect: true })
+  error = false;
+
+  override render() {
+    // Specify the component size
+
```
Comment (yodas7) [inline src/components/toggle/daikin-toggle.ts:129]:
```suggestion
```

File: src/components/toggle/daikin-toggle.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      class=${toggleClassName}
+      type="checkbox"
+      name=${this.name}
+      value=${this.value}
+      .checked=${this.toggled}
+      ?disabled=${this.disabled}
+      @change=${this._handleChange}
+    />`;
+
+    return html`${inputTag}`;
```
Comment (yodas7) [inline src/components/toggle/daikin-toggle.ts:142]:
```suggestion
    return html`<input
      class=${cvaToggle({ size: this.size })}
      type="checkbox"
      name=${this.name}
      value=${this.value}
      .checked=${this.toggled}
      ?disabled=${this.disabled}
      @change=${this._handleChange}
    />`;
```

**Review by yodas7 — COMMENTED** [review]
Some nitpicking comments.

File: src/components/toggle/daikin-toggle.ts [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        ],
+      },
+    },
+  }
+);
+
+type ToggleVariantProps = MergeVariantProps<typeof cvaToggle>;
+
+/**
+ * Primary UI component for user interaction
```
Comment (yodas7) [inline src/components/toggle/daikin-toggle.ts:59]: I would like to add [lit-analyzer friendly JSDoc](https://github.com/runem/lit-analyzer/blob/master/docs/readme/jsdoc.md).
(`@element`, `@attr` and `@prop` are not needed as they're inferred from individual prop definitions.)

```suggestion
 * A toggle switch component.
 *
 * @fires change - Emitted when the toggle switch is toggled.
```

**Review by yodas7 — APPROVED** [review]
LGTM! Thanks.

**Review by hiro-daikin — APPROVED** [review]

**Pull request merged**